### PR TITLE
Integrate new ribbon

### PR DESCRIPTION
**Pull request by nathandunn**
Being done in conjunction with https://agr-jira.atlassian.net/browse/AGR-860

### CODE REVIEW

**Comment by nathandunn** [comment]
Ugh, merge ready for Mondaty

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "url": "https://github.com/alliance-genome/agr_ui.git"
   },
   "dependencies": {
-    "@sibyl229/gene-ontology-ribbon": "1.2.7",
+    "@nathandunn/gene-ontology-ribbon": "^1.2.8",
```
Comment (sibyl229) [inline package.json]: Please use exact version to avoid unexpected incompatibility :)

File: src/containers/genePage/index.js [hunk 2]
```diff
@@ -97,7 +97,8 @@ class GenePage extends Component {
         </Subsection>
 
         <Subsection title='Function – GO Annotations'>
-          <GeneOntologyRibbon id={data.primaryId} />
+          <GeneOntologyRibbon id={data.primaryId} slim='agr' />
+          {/*<GeneOntologyRibbon id={this.props.data.primaryId} slim='agr' />*/}
```
Comment (sibyl229) [inline src/containers/genePage/index.js]: Maybe some commented out lines needs to be removed before pulling into develop?

File: webpack.config.babel.js [hunk 3]
```diff
@@ -10,9 +10,8 @@ let API_URL = process.env.API_URL || 'http://localhost:8080';
 let DEV_SERVER_UI_PORT = process.env.DEV_SERVER_UI_PORT || '2992';
 let JBROWSE_URL = process.env.JBROWSE_URL || 'http://jbrowse.alliancegenome.org';
 let JBROWSE_PORT = process.env.JBROWSE_PORT || '8891';
-let MANET_URL = process.env.MANET_URL || 'http://jbrowse.alliancegenome.org';
-let MANET_PORT = process.env.MANET_PORT || '8891';
-let LIVE_UI = process.env.LIVE_UI || 'false';
+//let LIVE_UI = process.env.LIVE_UI || 'false';
+let LIVE_UI = 'true' ;
```
Comment (sibyl229) [inline webpack.config.babel.js]: Is this intentional?

Comment (nathandunn) [reply webpack.config.babel.js]: the MANET part is, not sure what the LIVE_UI is, or why I commented it out.

Comment (nathandunn) [reply webpack.config.babel.js]: I'll create a PR to put the LIVE_UI back

**Comment by nathandunn** [comment]
@sibyl229   Shoot, I only saw your last comment.   I'll try to clean some of the comments out as well.

**Comment by sibyl229** [comment]
@nathandunn no problem. I know you will have another PR coming through when you sort out the npm stuff.

**Pull request merged**